lib/analyze-scope.js

```diff
@@ -18,6 +18,10 @@ const flowFlippedAliasKeys = t.FLIPPED_ALIAS_KEYS.Flow.concat([
   "ObjectPattern",
   "RestElement",
 ]);
+
+// TODO Remove!
+t.VISITOR_KEYS.CallExpression.push("typeArguments");
```

Comment:
I'll remove this once my other PR in Babel gets merged and released. Please do not merge this yet as it is now.